sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/KnownMethods.cs

```diff
@@ -245,6 +245,16 @@ public static bool IsListAddRange(this IMethodSymbol methodSymbol)
                 methodSymbol.ContainingType.ConstructedFrom.Is(KnownType.System_Collections_Generic_List_T);
         }
 
+        public static bool IsProbablyEventHandler(this IMethodSymbol methodSymbol)
+        {
+            return methodSymbol.ReturnsVoid &&
+                methodSymbol.Parameters.Length == 2 &&
+                methodSymbol.Parameters[0].Type.Is(KnownType.System_Object) &&
+                // Cannot check that the 2nd parameters is or derives from EventArgs as UWP uses other kind of event args
+                // (e.g. ILeavingBackgroundEventArgs)
+                methodSymbol.Parameters[1].Type.ToString().EndsWith("EventArgs", StringComparison.Ordinal);
```

Comment:
I guess we could check that the current class ends with _EventArgs_ or one of its parent classes does.
Is it worth it?